### PR TITLE
Add estimated distances to Delivery Trip

### PR DESCRIPTION
**Pull request by Alchez**
Added to existing PR upstream: https://github.com/frappe/erpnext/pull/15566

<hr>

- Google Maps returns distance values in meters, so we pick that up, convert it into the system-default distance UOM, and store it in both the Delivery Trip and the Delivery Stop documents.
- The destination's latitude and longitude are also stored in hidden fields inside the Delivery Stop document.

<hr>

**Global Defaults**

image

**Parent doc**

image

**Child doc**

image

### CODE REVIEW

File: erpnext/setup/doctype/global_defaults/global_defaults.json [hunk 1]
```diff
@@ -1,210 +1,291 @@
...
    "unique": 0
   }, 
   {
+   "allow_bulk_edit": 0, 
+   "allow_in_quick_entry": 0, 
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
+   "columns": 0, 
+   "default": "Kilometer",
```
Comment (vjFaLk) [inline erpnext/setup/doctype/global_defaults/global_defaults.json]: If you're setting a default here then we need to write a patch to do the same for existing sites.

File: erpnext/stock/doctype/delivery_stop/delivery_stop.json [hunk 2]
```diff
@@ -525,6 +557,168 @@
...
+   "unique": 0
+  }, 
+  {
+   "allow_bulk_edit": 0, 
+   "allow_in_quick_entry": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "more_information_section",
```
Comment (vjFaLk) [inline erpnext/stock/doctype/delivery_stop/delivery_stop.json:697]: Is this on the bottom?

**Review by vjFaLk — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/delivery_stop/delivery_stop.json (hunk 2 above)
Comment (Alchez) [reply erpnext/stock/doctype/delivery_stop/delivery_stop.json:697]: Yeah, at the very end.

File: erpnext/setup/doctype/global_defaults/global_defaults.json (hunk 1 above)
Comment (Alchez) [reply erpnext/setup/doctype/global_defaults/global_defaults.json]: This was from a previous test. I've removed it now.

**Review by vjFaLk — APPROVED** [review]

**Pull request merged**